build/_k-standard-goals.shade

```diff
@@ -88,19 +90,29 @@ default Quiet='${ false }'
 
 #build-compile target='compile' if='Directory.Exists("src")'
   @{
-    var projectFiles = Files.Include("src/**/project.json").ToList();
-    if (ShouldRunInParallel)
+    if (IsBuildV2) 
     {
-        Parallel.ForEach(projectFiles, projectFile => DnuPack(projectFile, BUILD_DIR, Configuration));
+        const string buildPattern = "src/**";
+        
+        Console.WriteLine("Running new build");
```

Comment:
Remove since information is obvious from logging of the actual `build` and `pack` commands.